torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -132,6 +132,16 @@ def parallelize_llama(
             enable_compile=job_config.training.compile,
             enable_compiled_autograd=job_config.experimental.enable_compiled_autograd,
         )
+    elif parallel_dims.cp_enabled:  # CP is enabled without DP
```

Comment:
Can we consolidate this case with the FSDP case? It looks only the mesh assignment part is different, the `apply_fsdp` part is the same.